other/tests.py

```diff
@@ -60,12 +61,14 @@ def setUp(self):
         self.client = APIClient()
         self.client.force_authenticate(self.user)
 
+    @sync_to_async
```

Comment:
This annotation won't work. It's preventing the test from being called altogether.